CRM/Contact/BAO/Relationship.php

```diff
@@ -2076,11 +2076,11 @@ public static function getContactRelationshipSelector(&$params) {
       // FIXME: we cannot directly determine total permissioned relationship, hence re-fire query
       $permissionedRelationships = CRM_Contact_BAO_Relationship::getRelationship($params['contact_id'],
         $relationshipStatus,
-        0, 0, 0,
+        0, 1, 0,
         NULL, NULL,
         $permissionedContacts
       );
-      $params['total'] = count($permissionedRelationships);
+      $params['total'] = $permissionedRelationships;
```

Comment:
As `$permissionedRelationships` variable isn't used elsewhere in the code, can we directly assign `$params['total']` initially(line 2077) instead of assigning to a new variable in between?